contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -103,8 +99,9 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, ERC677Recei
   /**
    * @inheritdoc IFunctionsSubscriptions
    */
-  function getMaxConsumers() external pure override returns (uint16) {
-    return MAX_CONSUMERS;
+  function getMaxConsumers() public view override returns (uint16) {
```

Comment:
Do we need this if we already  have a getter exposed for retrieving the whole config?

Reply:
Make this into the internal virtual function